registry/tracecat_registry/templates/detect/lookup_user_by_email/okta.yml

```diff
@@ -30,4 +30,5 @@ definition:
           Authorization: "SSWS ${{ SECRETS.okta.OKTA_API_TOKEN }}"
         params:
           search: ${{ FN.url_encode(steps.search_query.result) }}
-  returns: ${{ steps.search_user_by_email.result[0] }}
+  # core.http_request returns data, headers, and status_code
+  returns: ${{ steps.search_user_by_email.result.data }}
```

Comment:
Do we still want to grab the first result only? @topher-lo

Reply:
Yeah there should only be one user